helm/values.yaml

```diff
@@ -35,6 +35,22 @@ nginx-ingress:
       enabled: true
     service:
       type: LoadBalancer
+prometheus-operator:
+  enabled: true
```

Comment:
I think the local presets file should toggle this off by default, as you probably don't want prometheus on Minikube.

Reply:
if you don't think it would be a useful for gathering relative performance changes in a local environment i can get behind that. it doesn't really add much (not like the elasticsearch cluster for logging) but if it's not useful then it should get toggled off.